### ISSUE TITLE
An option to split coins (double currency) in Mint

### ISSUE THREAD

**Issue by Elliander** [StrangeLoopGames/EcoSuggestions#1025]
In the mint, when creating a currency, there is a setting to decide how many coins are backed by a given item. For example, 100 coins per bar of gold. This basically means that one bar of gold is worth 100 coins and this cannot be changed for obvious reasons.

However, in the real world, Stock Doubling is a concept. It basically means that everyone with any amount of the stock now have twice as much, but each share is worth half as much. For them there is no change in value, just a change in numbers.

For a currency, suppose the supply I create sees utilization, but maybe it's difficult to set prices for items with a small amount. At present the only solution would be to create more coins, but that is seen as inflation.

An ideal solution would be if there is a setting to actually adjust how many coins equal a bar of gold, and for everyone's holdings to be adjusted to reflect that. Shop prices should be adjusted automatically at the same time to prevent exploitation.

In practice, it would mean that if I produce a thousand coins and distribute it evenly and after a while it has widespread use, but more coins are needed, I could go to the mint, and double how many coins a single bar of gold is worth, and then everyone will see their coins double and all shop prices that are based on that coin double with it, so there is no change in what anyone has, it just solves a potential problem of people having to set extreme fraction of a coin prices if there aren't enough coins in circulation.

**Comment by tobyhine** [comment]
Alternatively or additionally, the ability to turn currency back into its underlying material would be useful. Either to allow a complete reissue of a currency at a different backing ratio, or to allow a transition to an unbacked currency.

**Comment by mirasrael** [comment]
Nothing wrong in inflation actually.

**Comment by trickenso** [comment]
> Nothing wrong in inflation actually.

Agreed - most backed currencies in Eco could use more of it. Devaluation would reintroduce a useful real-world mechanism to allow more inflation. Without it, you're stuck with just mining more gold.

A few sources on the subject:

> Benefits Of Commodity Money: Lower Inflation
> Because commodity money is based on a physical resource, it is less prone to devaluing from inflation.

https://yourmoney.lumio-app.com/commodity-backed-money-vs-fiat-money/

> Generally, a devaluation is likely to contribute to inflationary pressures because of higher import prices and rising demand for exports.

https://www.economicshelp.org/macroeconomics/macroessays/does-devaluation-cause-inflation/

> There is strong empirical evidence of a direct relationship between the growth of the money supply and long-term price inflation

https://en.wikipedia.org/wiki/Money_supply

> Monetarists believe the most significant factor influencing inflation or deflation is how fast the money supply grows or shrinks.

https://en.wikipedia.org/wiki/Inflation#Monetarist_view

> Currency devaluations can be used by countries to achieve economic policy. Having a weaker currency relative to the rest of the world can help boost exports, shrink trade deficits and reduce the cost of interest payments on its outstanding government debts.

https://www.investopedia.com/articles/investing/090215/3-reasons-why-countries-devalue-their-currency.asp

The only way to inflate in the current system is to shift wholly to an unbacked currency or turn more gold into coins, hence the suggestion that more flexibility would be useful.

Backed currencies are more inflexible than fiat currencies, and that makes sense, but making the game's version even more inflexible than real-life backed currencies adds even more friction to their use, when they already have quite a lot.

It also removes all the fun of the currency being worth less than its underlying commodity:

https://commons.wikimedia.org/wiki/File:Bundesarchiv_Bild_102-00104,_Inflation,_Tapezieren_mit_Geldscheinen.jpg
